test/common/memory_usage.h

```diff
@@ -133,6 +130,16 @@ namespace utils {
 #endif
     }
 
+    std::size_t GetApplicationMemoryUsage(MemoryStatType stat = currentUsage) {
```

Comment:
What is the difference between `GetApplicationMemoryUsage` and `GetMemoryUsage`? Why it cannot be one function?